woocommerce/wc-admin
# Replace flag icon with component and svg background to png

This PR fixes the issue with build times becoming exceedingly slow when we use svg backgrounds included via scss by:

1. Changing `icon-flag.svg` svg to a component
2. Converting `wcpay-banner.svg` to a png image file

This issue may have been caused by one of webpack plugins trying to optimize the svg in question. The biggest culprit to this was caused by the file `images/wc_pay_welcome_page/wcpay-banner.svg` which size was around 150kb. I think it's much more optimal to use png instead for the particular file, as we're neither scaling the image nor dynamically modifying the svg's props. Size is reduced down to around 18kb.

### Detailed test instructions:

- Run `npm run start` and wait for it to finish the initial build
- Change anything small in js or scss to trigger rebuild
- Observe the rebuild time is less than 1 - 2 seconds

No changelog
